src/50_Visual_Effects/Basics_of_orientation_effects.html

```diff
@@ -0,0 +1,304 @@
+
+  <!--
+    ## Styles
+
+    The CSS used for these samples are included here for reference.
+
+    These rules are simply needed to make the samples work but are not fundamental
+    to the concepts covered here.
+  -->
+  <style amp-custom>
+    #scrollText {
+      font-family: Helvetica, sans-serif;
+      padding: 10px;
+      max-width: 412px;
+      height: 500px;
+      margin: auto;
+      overflow: hidden;
+    }
+
+    #ipsum {
+      background-image: linear-gradient(white, deeppink);
+    }
+
+    amp-img img {
+      object-fit: cover;
+    }
+
+    .clock-hand {
+      position: absolute;
+      width: 2vw;
+      height: 20vw;
+      max-height: 90px;
+      max-width: 10px;
+      top: 50%;
+      left: 50%;
+      z-index: 2;
+      background: #FAFAFA;
+      transform-origin: 50% 0%;
+      border-radius: 10px;
+      transform: rotate(-180deg)
+    }
+
+    .rotate-2 {
+      transform: rotate(-270deg)
+    }
+  </style>
+</head>
+
+<body>
+
+  <!--
+  ## Tilt based scroll animations
+
+  Let's create a tilt based scroll animation where you can scroll up 
+  and down on a page as you tilt a device along the y axis.
+
+  This sample showcases the core concept behind combining
+  `amp-orientation-observer` and `amp-animation`: The ability to progress
+  through a keyframe animation timeline as the device is tilted.
+  -->
+
+  <!--
+  Our scroll scene is a `div` with some Lorem Ipsum text in it. We add an
+  `amp-orientation-observer` element as child of the scene to monitor the device's position. 
+  Let's take a look at the details:
+  - **`on:beta`**: This event is triggered by orientation observer as deivce orientation 
+  along the `beta` axis changes. The event provides a percentage value
+  (decimal between 0 and 1) representing how far the device is within the `beta` axis.
+  - **`anim.seekTo(percent=event.percent)`**: We will define an `amp-animation`
+  that will do the spinning in the next step, here we are coupling `amp-orientation-observer` and
+  `amp-animation` by triggering a `seekTo` action on the animation as `beta` events occurs.
+  This is how we specify that we like to progress through the animation
+  timeline as the device moves along the `beta` axis.
+  - **`beta-range`**: Each axis has a range of degrees it tracks. 
+  For the `beta` axis the default range is -180 to 180 degrees. 
```

Comment:
Does it make sense to add something like: "In this sample we set the beta range to `0 180` as the text cannot scroll beyond it's first paragraph." ?

Reply:
Done.